.gitignore

```diff
@@ -0,0 +1 @@
+ci-bot
```

Comment:
Any reason why we should ignore ci-bot?

Reply:
Yep, the ci-bot binary file. we should not keep it in this repo.

Reply:
Agree with @edisonxiang